frontend/src/views/DiagnosticEditor/index.vue

```diff
@@ -70,7 +77,24 @@
           </v-row>
         </v-form>
 
-        <p class="caption grey--text text--darken-1" v-if="!hasActiveTeledeclaration">
+        <v-radio-group
+          v-model="diagnostic.centralKitchenDiagnosticMode"
+          :readonly="hasActiveTeledeclaration"
+          :disabled="hasActiveTeledeclaration"
+          class="mt-0"
+          v-if="isCentralCanteen"
+        >
+          <v-radio v-for="type in centralKitchenDiagnosticModes" :key="type.key" :label="type.label" :value="type.key">
+            <template v-slot:label>
+              <span class="grey--text text--darken-3">{{ type.label }}</span>
+            </template>
+          </v-radio>
+        </v-radio-group>
+
+        <p
+          class="caption grey--text text--darken-1"
+          v-if="!hasActiveTeledeclaration && (showApproPanel || showNonApproPanels)"
```

Comment:
`showApproPanel || showNonApproPanels` could be replaced by `showExpansionPanels`